test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,41 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+
+        //only one null object
+        assertContainsNull((Object) null);
+        assertContainsNull(null, "123", "abc");
+        assertContainsNull( Integer.valueOf(1), null);
+
+        //multiple null objects
+        assertContainsNull(null, null);
+        assertContainsNull(null, "123", "ABC", null );
+        assertContainsNull(null, 1, Integer.valueOf(1), null);
+
+        //no null objects
+        assertContainsNoNull("123");
+        assertContainsNoNull("123", "abc","abc" );
+        assertContainsNoNull(1, Integer.valueOf(1));
+
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertContainsNull(Object... objects) {
```

Comment:
Please create each PR separately. You can try to exclude changes from previous weeks.